base/commands/shell.go

```diff
@@ -103,6 +105,33 @@ func (cm *ShellCommand) ExecInteractive(ctx context.Context, ec plug.ExecContext
 	return sh.Start(ctx)
 }
 
+func prompt(cfgPath string) string {
+	if cfgPath == "" {
+		return "CLC> "
+	}
+	// Best effort for absolute path
+	if !filepath.IsAbs(cfgPath) {
```

Comment:
Is this check necessary?

Reply:
It's not necessary, I will take the statemens inside to the root level